### ISSUE TITLE
'titlestring' not being reset

### ISSUE THREAD

**Issue by desjardinsm** [jeetsukumaran/vim-filebeagle#34]
I had noticed that after entering a FileBeagle buffer that `titlestring` was overridden and wasn't being reset.  I checked and although the previous string was being saved to `old_titlestring`, the line that would have reset it, at [autoload/filebeagle.vim#L695](https://github.com/jeetsukumaran/vim-filebeagle/blob/master/autoload/filebeagle.vim#L695), was commented out:

```
" if has("title")
"     let &titlestring = self.old_titlestring
" endif
```

Is there a reason for this?

**Comment by lexazloy** [comment]
+1

I love my `titlestring`. Why filebeagle override it? Looks like a bug.

**Comment by jeetsukumaran** [comment]
Not sure why this was commented out. No real reasons occurs to me. When I have time, I can try to look into if reinstating the code has problems.

**Comment by jeetsukumaran** [comment]
Fixed here: 7dd37c6.
